napari/utils/settings/_appearance.py

```diff
@@ -0,0 +1,51 @@
+from typing import Tuple, Union
+
+from pydantic import Field
+
+from ..events.evented_model import EventedModel
+from ..translations import trans
+from ._fields import SchemaVersion, Theme
+
+
+class AppearanceSettings(EventedModel):
```

Comment:
This does not need to inherit from BaseSettings now?

Reply:
correct... unless you want each subfield itself to have the capability to save/load, etc...  then only the main `NapariSettings` object would need all that stuff.
If we did someday want to move to a pattern where each sub-field had it's own file, then we could add back the `BaseSettings` inheritance no problem (I did quickly check to make sure...), but for now, might as well inherit from less

Reply:
Understood!